_test/test_sqw_file/test_faccess_sqw_v3.m

```diff
@@ -287,6 +288,44 @@ function test_serialize_deserialise_faccess(obj)
             
             assertEqual(fo,fr);
         end
+        %
+        function test_wrong_file_name_activated(obj)
+            ld = sqw_formats_factory.instance.get_loader(obj.sample_file);
+            sample_obj = ld.get_sqw();
+            
+            test_name = 'test_wrong_file_name_activated_1.sqw';
+            targ_file = fullfile(tmp_dir(),test_name);
+            
+            wrt =sqw_formats_factory.instance.get_pref_access(sample_obj);
+            wrt = wrt.init(sample_obj,targ_file);

```

Comment:
may be the code is not fully completed. the first row returns empty accessor and the second one initializes it. May be we should re-think faccess interface. It will be very timely operation when we we start writing hdf accessor, but it does not have any relation to this ticket. I've used the existing interface as it is.